src-tauri/src/cloud.rs

```diff
@@ -0,0 +1,33 @@
+use crate::models::{CloudTest, CloudTestListResponse};
+
+fn get_request(url: &str, cloud_token: &str) -> reqwest::RequestBuilder {
+
+    // NOTE: messy but will do
+    let client = reqwest::Client::new();
+    let url = format!("https://api.k6.io{}", url);
+    client.get(url)
+        .header(reqwest::header::AUTHORIZATION, format!("Token {}", cloud_token))
+        .header(reqwest::header::USER_AGENT, "kroco6")
+}
```

Comment:
I was planning to use the same client/headers setup for multiple cloud requests in the future, I went as far as letting it understand the type you wanted as to handle the complete request inside of it but I encountered issues not worth pursuing at this stage... so it just creates the best request & adds headers for now.